### DISCUSSION TITLE
Blocked wallet addresses

### DISCUSSION THREAD

**Discussion by ag-msq** [snapshot-labs/snapshot#3869] [Q&A]
Hello!

Guys, thank you for developing such a great product!

We are integrating our service with Snapshot and have one unclear issue. For example, we define that user should have voting power if he has verified email in our system. First user hasn't fill out the email and go to Snapshot, it requests our site to define voting power, it answers negative. So user cannot vote, this is correct behaviour. But after that user has specified his email and goes to Snapshot with the same wallet, and Snapshot doesn't ask our side about the voting power at all.

As we understand this is some kind of ban for wallets having not enough voting power. But is there an option for them to re-request our platform to refresh information about voting power?

Thank you!

**Comment by Shoaib9990** [comment]
Gud

**Comment by Shoaib9990** [comment]
Gud thinking
